generators/app/templates/src/app/components/CustomTextInput/interface.ts

```diff
@@ -0,0 +1,16 @@
+export interface CustomTextInputProps extends TextInputProps, TextProps {
+  bottomBorder?: boolean;
+  multiline?: boolean;
+  textStyles?: TextStyle;
+  onBlur?: TextInputProps["onBlur"];
+  onChange?(e: any): any;
+  onFocus?: TextInputProps["onFocus"];
+  value?: string;
+  placeholderTextColor?: string;
+  showEye?: boolean;
+  secureTextEntry?: boolean;
+  style?: ViewProps["style"];
+  autoCompleteType?: TextInputProps["autoCompleteType"];
+  title?: string;
+  titleStyles?: TextStyle;
```

Comment:
Also, I think we should use only one interface for both components because they are repeated. What  do you think?